dex-it/src/test/scala/com/wavesplatform/it/sync/networking/MultipleMatchersOrderCancelTestSuite.scala

```diff
@@ -0,0 +1,100 @@
+package com.wavesplatform.it.sync.networking
+
+import java.nio.charset.StandardCharsets
+import java.util.concurrent.ThreadLocalRandom
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.dex.domain.account.KeyPair
+import com.wavesplatform.dex.domain.asset.Asset
+import com.wavesplatform.dex.domain.asset.Asset.Waves
+import com.wavesplatform.dex.domain.bytes.ByteStr
+import com.wavesplatform.dex.domain.order.OrderType
+import com.wavesplatform.dex.it.api.responses.dex.OrderStatus
+import com.wavesplatform.dex.it.docker.DexContainer
+import com.wavesplatform.it.MatcherSuiteBase
+import com.wavesplatform.it.tags.DexItKafkaRequired
+import org.scalatest
+
+@DexItKafkaRequired
+class MultipleMatchersOrderCancelTestSuite extends MatcherSuiteBase {
+
+  private val kafkaServers = "kafka-dev-aws-fr-1.wvservices.com:9092"
```

Comment:
Please do not commit the code with our configuration. Use `System.getenv("KAFKA_SERVER")` (see `HasDex`).

Reply:
Done